lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -58,7 +58,10 @@ sub ialist_json :Chained('base') :PathPart('json') :Args() {
 
     my @ial = $rs->search(
         {'topic.name' => { '!=' => 'test' },
-         'me.dev_milestone' => { '=' => 'live'}
+         -or => [
```

Comment:
I can guess what this construct does, but this ` -or ` syntax.. what does it mean?